aiofluent/sender.py

```diff
@@ -96,16 +98,23 @@ def lock(self):
 
     async def get_writer(self):
         async with self.lock:
-            if self._writer is not None:
-                return self._writer
+
+            # pool is fully allocated, return from the pool
+            if len(self._pool_conn) == self._size_pool_conn:
```

Comment:
Maybe just fill pool up with all the connections immediately and then just use `random.choice`? It would make the code more simple.